### DISCUSSION TITLE
Multi-store all context update

### DISCUSSION THREAD

**Discussion by ysnmc** [PrestaShop/PrestaShop#36720] [Q&A] [answered]
Hi all! I have a problem with multistore, when I want to update the prices while all stores are selected, the descriptions from my default website also go to the other websites, but each one needs to have a different description. So how can I change only the price while all stores are selected?

**Comment by kpodemski** [comment, marked as answer]
Do you use PrestaShop 1.7? If so, if you use multi-store, I highly recommend testing PrestaShop 8.1 and the new product page, which eases how you manage your products in multi-store. You can explicitly select which fields you want to update globally and which do not.

**Reply by ysnmc** [reply]
Thank you so much for your reply, yes I was using PrestaShop 1.7 and I'm gonna update to 8.1.
